tests/Functional/FactoryDoctrineCascadeTest.php

```diff
@@ -119,17 +125,17 @@ public function many_to_many_reverse_relationship(): void
      */
     public function one_to_one_relationship(): void
     {
-        $variant = factory(Variant::class, [
+        $product = factory(Product::class, [
```

Comment:
had to change this test, because it was not anymore relying on a `OneToOne` + cascade persist